terminal/output.go

```diff
@@ -111,15 +120,19 @@ func (terminal *Terminal) processInput(pty chan rune) {
 
 		b = <-pty
 
+		// debug += fmt.Sprintf("0x%x ", b)
```

Comment:
remove?

Reply:
Left here intentionally so that I can turn on stream dump without writing it over again.